dotcom-rendering/src/palette.ts

```diff
@@ -902,6 +903,163 @@ const standfirstTextDark = ({
 	}
 };
 
+const cardBorderTopLight = (format: ArticleFormat): string => {
+	return decidePalette(format).topBar.card;
+};
+const cardBorderTopDark = (): string => {
+	return sourcePalette.neutral[20];
+};
+const cardAgeTextLight = (format: ArticleFormat): string => {
+	return decidePalette(format).text.cardFooter;
+};
+const cardAgeTextDark = (): string => {
+	return sourcePalette.neutral[60];
+};
+const cardBackgroundLight = (format: ArticleFormat): string => {
+	return decidePalette(format).background.card;
+};
+const cardBackgroundDark = ({ design, theme }: ArticleFormat): string => {
+	switch (design) {
+		case ArticleDesign.LiveBlog:
+			switch (theme) {
+				case Pillar.Lifestyle:
+					return sourcePalette.lifestyle[100];
+				case Pillar.Sport:
+					return sourcePalette.sport[100];
+				case Pillar.Culture:
+					return sourcePalette.culture[100];
+				case Pillar.Opinion:
+					return sourcePalette.opinion[100];
+				case ArticleSpecial.Labs:
+					return sourcePalette.labs[200];
+				case ArticleSpecial.SpecialReport:
+					return sourcePalette.specialReport[100];
+				case Pillar.News:
+				default:
```

Comment:
I think for `theme` we should be explicit and handle all possible cases rather than relying on a `default` case.